babel.config.js

```diff
@@ -32,6 +32,7 @@ module.exports = (api) => {
     ];
     const plugins = [
         path.join(nodePath, '@babel/plugin-proposal-class-properties'),
+        path.join(nodePath, '@babel/plugin-transform-runtime'),
```

Comment:
In order to use `async` and `await`, I had to add this plugin, following Babel's recommendation https://babeljs.io/docs/en/babel-plugin-transform-runtime. 

Another way to fix it, which is how I fixed it in the tests, was to include `regenerator-runtime` directly in the file (https://www.npmjs.com/package/regenerator-runtime), but I read a few posts that said the Babel way was preferred to avoid polluting the global scope. 

Note: I originally incorrectly thought the `regenerator-runtime` error was caused by using `fetch` and Jest, but it turned out to be an issue with using `async` and `await`.